### PR TITLE
Several fixes in Using Web Workers

### PR DESCRIPTION
**Pull request by mfuji09**
> What was wrong/why is this fix needed? (quick summary only)

- replace &nbsp; with normal space
- fix some spacings
- fix some markups

> MDN URL of main page changed

https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers

### CODE REVIEW

File: files/en-us/web/api/web_workers_api/using_web_workers/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 ---
 <div>{{DefaultAPISidebar("Web Workers API")}}</div>
 
-<p><span class="seoSummary">Web Workers are a simple means for web content to run scripts in background threads. The worker thread can perform tasks without interfering with the user interface. In addition, they can perform I/O using <code><a class="internal" href="/en-US/docs/Web/API/XMLHttpRequest">XMLHttpRequest</a></code> (although the <code>responseXML</code> and <code>channel</code> attributes are always null) or <code><a href="/en-US/docs/Web/API/Fetch_API">fetch</a></code> (with no such restrictions). Once created, a worker can send messages to the JavaScript code that created it by posting messages to an event handler specified by that code (and vice versa).</span> This article provides a detailed introduction to using web workers.</p>
```
Comment (s-mrb) [inline files/en-us/web/api/web_workers_api/using_web_workers/index.html:15]: Can you please explain why this change is necessary? Even without change it looks fine on website.

Comment (s-mrb) [reply files/en-us/web/api/web_workers_api/using_web_workers/index.html:15]: I got it. :)

File: files/en-us/web/api/web_workers_api/using_web_workers/index.html [hunk 2]
```diff
@@ -536,7 +536,7 @@ <h3 id="Passing_data_by_transferring_ownership_transferable_objects">Passing dat
 </pre>
 
 <div class="note">
-<p><strong>Note</strong>: For more information on transferable objects, performance, and feature-detection for this method, read <a href="http://updates.html5rocks.com/2011/12/Transferable-Objects-Lightning-Fast">Transferable Objects: Lightning Fast!</a> on HTML5 Rocks.</p>
```
Comment (s-mrb) [inline files/en-us/web/api/web_workers_api/using_web_workers/index.html:539]: Nice catch, it should be changed.

I am curious why/how so many changes in this commit whereas only things that is actually changed changed is this `http` to `https` conversion, and 2 more?

**Review by sideshowbarker — APPROVED** [review]
